__init__.py

```diff
@@ -78,6 +80,32 @@ def gbp():
     return url
 
 
+def ft():
+    """Custom news fetcher for today's FT news briefing"""
+    url = 'https://www.ft.com/newsbriefing'
+    page = urlopen(url)
+
+    # Use bs4 to parse website and get mp3 link
+    soup = BeautifulSoup(page, features='html.parser')
+    result = soup.find('time')
+
+    # Get today and yesterday's date to match div and play most recent news
+    today = datetime.date.today()
```

Comment:
Hey thanks for updating and rebasing.

In my previous comment I didn't take into account that we were already importing datetime from datetime which means this is trying to access `datetime.datetime.date.today()`
So just need to switch this to:
```
today = date.today()
```
and then add `date` to the import at the top:
```
from datetime import date, datetime, timedelta
```

Reply:
My bad, that's all done up now!